tools/parameter_tuning.py

```diff
@@ -0,0 +1,55 @@
+from sisyphus import Job, Task, tk
+from typing import Any, Literal, Sequence, Union
+
+import numpy as np
+
+from i6_core.util import instanciate_delayed
+
+
+class PickOptimalParametersJob(Job):
```

Comment:
```suggestion
class WriteBestParameterVariablesJob(Job):
```
?

My main wish would be to have "Variable" somewhere inside the name (or sth similar to reflect that this writes Sisyphus variables as output). Because this was the main thing I got confused about.